app/src/main/java/com/babylon/wallet/android/domain/model/SecurityProblem.kt

```diff
@@ -3,22 +3,28 @@ package com.babylon.wallet.android.domain.model
 // refer to this table
 // https://radixdlt.atlassian.net/wiki/spaces/AT/pages/3392569357/Security-related+Problem+States+in+the+Wallet
 sealed interface SecurityProblem {
+
     data class EntitiesNotRecoverable(
         val accountsNeedBackup: Int,
-        val personasNeedBackup: Int
+        val personasNeedBackup: Int,
+        val hiddenAccountsNeedBackup: Int,
+        val hiddenPersonasNeedBackup: Int
```

Comment:
because the requirements are constantly changing I decided to have all the amounts of the non-hidden and hidden accounts in the data class, and then in the viewmodels we build the logic